edge-util/src/Microsoft.Azure.Devices.Edge.Util/Option.cs

```diff
@@ -140,8 +140,22 @@ public void ForEach(Action<T> action)
             }
         }
 
+        public void ForEach(Action<T> action, Action none = null)
+        {
+            if (this.HasValue)
+            {
+                action(this.Value);
+            }
+            else
+            {
+                none?.Invoke();
```

Comment:
Hmm, the idea of Option is to avoid nulls.. so maybe we should avoid one here. :) Separate overload maybe?

Reply:
Yes, after re-reviewing this my suggestion is a bit ironic. I like `ForEach` syntax better, but if an overload function works, that's how it should be implemented.